runelite-client/src/main/resources/npc_health.json

```diff
@@ -1160,5 +1160,6 @@
 	"Alchemical Hydra_426": 1100,
 	"Undead Druid_105": 140,
 	"Temple Spider_75": 70,
-	"Sarachnis_318": 400
+	"Sarachnis_318": 400,
+	"Zombie_44": 40
```

Comment:
I know not everything is organized in this file, but let's keep this with the other zombies already defined in this file, around line 9.

Reply:
Thanks. I grouped the zombies together.